schema/deploy/computed_columns/form_change_payment_percentage.sql

```diff
@@ -0,0 +1,18 @@
+-- Deploy cif:functions/form_change_payment_percentage to pg
+-- requires: tables/form_change
+-- 100 – ((-1.5) x GHG Emission Intensity Performance + 145)
+
+begin;
+
+create or replace function cif.form_change_payment_percentage(fc cif.form_change)
+returns numeric as
+$fn$
+
+  select
+    round(100 - ((-1.5) * (fc.new_form_data->>'adjustedEmissionsIntensityPerformance')::numeric + 145), 2);
```

Comment:
I might've missed there - where exactly does it show this logic?

Reply:
It's not super explicit; I'm getting it from this line under GHG Emission Intensity Performance (%) (Adjusted): `If this is filled out, use this value for further calculations`. 

@nanyangpro, could you please confirm how to handle calculated vs. adjusted GHG Emission Intensity Performance in this formula? Payment Percentage of Performance Milestone Amount  = 100 – ((-1.5) x GHG Emission Intensity Performance + 145) 

If we should be coalescing, we just ran into something similar (having to add a calculated value to the `newFormData`) with the release bugs. I ~*think you'll have to do the same thing here because it looks like there's a column for payment percentage in the `cif.emission_intensity_payment_percent` table. @dleard, would love your thoughts on this. If the payment percentage doesn't need to be added to any tables, then would it be all right to just coalesce `new_form_data->>'adjustedEmissionsIntensityPerformance` and `cif.emission_intensity_report_calculated_ei_performance(cif.emission_intensity_report)`? Or is that still one behind?